### PR TITLE
feat: support `simplecache` url chaining

### PR DESCRIPTION
**Pull request by lobis**
related to https://github.com/scikit-hep/uproot5/pull/1075.

I would like to add a test in uproot where we test the `simplecache` but the files I know of are too big, would it be possible to have a small test file hosted over xrootd so we can test it? (file would need to be downloaded whole).

Edit: At the end I ended up adding an xrootd server fixture to uproot so we can test this on local files, so no need to add a new small file, but I don't think it'll hurt to add one in any case.

### CODE REVIEW

**Comment by lobis** [comment]
After merging this PR a release would be needed in order to merge https://github.com/scikit-hep/uproot5/pull/1075.

File: src/fsspec_xrootd/xrootd.py [hunk 1]
```diff
@@ -412,6 +412,47 @@ async def _cat_file(self, path: str, start: int, end: int, **kwargs: Any) -> Any
                 self.timeout,
             )
 
+    async def _get_file(
+        self, rpath: str, lpath: str, chunk_size: int = 8192, **kwargs: Any
```
Comment (nsmith-) [inline src/fsspec_xrootd/xrootd.py:416]: Any particular reason for 8192?

File: src/fsspec_xrootd/xrootd.py [hunk 2]
```diff
@@ -412,6 +412,47 @@ async def _cat_file(self, path: str, start: int, end: int, **kwargs: Any) -> Any
...
+    ) -> None:
+        # Open the remote file for reading
+        remote_file = client.File()
+
+        try:
+            status, _n = await _async_wrap(
+                remote_file.open,
+                self.protocol + "://" + self.storage_options["hostid"] + "/" + rpath,
```
Comment (nsmith-) [inline src/fsspec_xrootd/xrootd.py:424]: Would `unstrip_protocol` be usable here?

**Review by nsmith- — APPROVED** [review]
Some minor suggestions.

**Comment by nsmith-** [comment]
Will this close #42 ?

File: src/fsspec_xrootd/xrootd.py (hunk 1 above)
Comment (lobis) [reply src/fsspec_xrootd/xrootd.py:416]: No particular reason, feel free to suggest any other value.

**Comment by lobis** [comment]
> Will this close #42 ?

Closes https://github.com/CoffeaTeam/fsspec-xrootd/issues/41

File: src/fsspec_xrootd/xrootd.py (hunk 1 above)
Comment (nsmith-) [reply src/fsspec_xrootd/xrootd.py:416]: It seems really small to me: consider a 50MB/s download: that's 6k chunks/s. We would be using a lot of CPU just to handle the chunks with python. I would choose something more like 256kiB-1MiB

Comment (lobis) [reply src/fsspec_xrootd/xrootd.py]: Makes sense, I'll increase the chunk size to 256 kiB

**Comment by nsmith-** [comment]
Go ahead and merge, also feel free to make a release

**Pull request merged**